sapl/base/views.py

```diff
@@ -915,6 +918,1229 @@ def get(self, request, *args, **kwargs):
+             )
+        )
+        tabela.append(
+            ('bancada_comissao_autor_externo',
+             'Bancadas e Comissões com autor externo',
+             len(bancada_comissao_autor_externo())
+             )
+        )
+
+        return tabela
+
+
+def bancada_comissao_autor_externo():
+    lista_bancada_autor_externo = []
+    lista_comissao_autor_externo = []
+
+    tipo_autor_externo = TipoAutor.objects.filter(descricao='Externo')
+
+    for bancada in Bancada.objects.all().order_by('nome'):
+        autor_externo = bancada.autor.filter(tipo=tipo_autor_externo)
+
+        # if len(autor_externo) == 1:
+
+        if len(autor_externo) > 0:
+            q_autor_externo = bancada.autor.get(tipo=tipo_autor_externo)
+            lista_bancada_autor_externo.append(
+                (q_autor_externo, bancada, 'Bancada', 'sistema/bancada')
+            )
+
+        # elif len(autor_externo) > 1:
+        #    q_autor_externo = bancada.autor.get(tipo=tipo_autor_externo)
+        #    for autor in q_autor_externo:
+        #        lista_bancada_autor_externo.append(
+        #            (q_autor_externo, bancada, 'Bancada')
+        #        )
+
+    for comissao in Comissao.objects.all().order_by('nome'):
+        autor_externo = comissao.autor.filter(tipo=tipo_autor_externo)
+
+        # if len(autor_externo) == 1:
+
+        if len(autor_externo) > 0:
+            q_autor_externo = comissao.autor.get(tipo=tipo_autor_externo)
+            lista_comissao_autor_externo.append(
+                (q_autor_externo, comissao, 'Comissão', 'comissao')
+            )
+
+        # elif len(autor_externo) > 1:
+        #     q_autor_externo = comissao.autor.get(tipo=tipo_autor_externo)
+        #     for autor in q_autor_externo:
+        #         lista_comissao_autor_externo.append(
+        #             (q_autor_externo, comissao, 'Comissão')
+        #         )
+
+    return lista_bancada_autor_externo + lista_comissao_autor_externo
+
+
+class ListarBancadaComissaoAutorExternoView(PermissionRequiredMixin, ListView):
+    model = get_user_model()
+    template_name = 'base/bancada_comissao_autor_externo.html'
+    context_object_name = 'bancada_comissao_autor_externo'
+    permission_required = ('base.list_appconfig',)
+    paginate_by = 10
+
+    def get_queryset(self):
+        return bancada_comissao_autor_externo()
+
+    def get_context_data(self, **kwargs):
+        context = super(
+            ListarBancadaComissaoAutorExternoView, self
+            ).get_context_data(**kwargs)
+        paginator = context['paginator']
+        page_obj = context['page_obj']
+        context['page_range'] = make_pagination(
+            page_obj.number, paginator.num_pages)
+        context[
+            'NO_ENTRIES_MSG'
+            ] = 'Nenhum encontrado.'
+        return context
+
+
+def autores_duplicados():
+    autores = {}
+    for a in Autor.objects.all().order_by('nome'):
+        key = "{}".format(a.nome)
```

Comment:
Não é necessário.

Reply:
Reescrevi essa função seguindo uma orientação anterior.
 
```suggestion
def autores_duplicados():
    lista_duplicados = []
    autores_duplicados = Autor.objects.values(
        'nome', 'tipo__descricao').annotate(
            count=Count('nome')).filter(count__gt=1)

    for autor in autores_duplicados:
        lista_duplicados.append(autor.values())

    return lista_duplicados
```